### PR TITLE
[DCSFUS-27] Develop an E-commerce Store Database

### PR DESCRIPTION
**Pull request by joejoba**

### CODE REVIEW

File: submissions/josephOlasimbo/database/init.sql [hunk 1]
```diff
@@ -0,0 +1,48 @@
+-- CREATE THE food bag DB
+CREATE DATABASE food_bag;
+USE food_bag;
+
+-- CREATE THE food TABLE
+CREATE TABLE food (
+	id INT NOT NULL AUTO_INCREMENT,
+    PRIMARY KEY (id)
```
Comment (ToluEnny24) [inline submissions/josephOlasimbo/database/init.sql:8]: this table shoud contain type too with the datatype as ENUM NOT NULL with the four type food categories

File: submissions/josephOlasimbo/database/init.sql [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+
+-- CREATE THE vendor food TABLE
+CREATE TABLE vendor_food (
+	id INT NOT NULL AUTO_INCREMENT,
+	admin_id INT NOT NULL,
+	food_id INT NOT NULL,
+	name VARCHAR(100) NOT NULL,
+	amount DECIMAL(10, 2) NOT NULL,
+    PRIMARY KEY (id)
+);
```
Comment (ToluEnny24) [inline submissions/josephOlasimbo/database/init.sql:19]: There should be FOREIGN KEYS referencing your admin_id and food_id

File: submissions/josephOlasimbo/database/init.sql [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+	id INT NOT NULL AUTO_INCREMENT,
+    PRIMARY KEY (id)
+);
+
+-- CREATE THE vendor food TABLE
+CREATE TABLE vendor_food (
+	id INT NOT NULL AUTO_INCREMENT,
+	admin_id INT NOT NULL,
+	food_id INT NOT NULL,
+	name VARCHAR(100) NOT NULL,
```
Comment (ToluEnny24) [inline submissions/josephOlasimbo/database/init.sql:16]: name should allow more characters. make it 250

File: submissions/josephOlasimbo/database/init.sql [hunk 4]
```diff
@@ -0,0 +1,48 @@
...
+	amount DECIMAL(10, 2) NOT NULL,
+    PRIMARY KEY (id)
+);
+
+-- CREATE THE admins TABLE
+CREATE TABLE admins (
+	id INT NOT NULL AUTO_INCREMENT,
+	name VARCHAR(255) NOT NULL,
+	type INT NOT NULL,
```
Comment (ToluEnny24) [inline submissions/josephOlasimbo/database/init.sql:25]: type should be ENUM 0 or 1

File: submissions/josephOlasimbo/database/init.sql [hunk 5]
```diff
@@ -0,0 +1,48 @@
...
+    PRIMARY KEY (id)
+);
+
+-- CREATE THE customers TABLE
+CREATE TABLE customers (
+	id INT NOT NULL AUTO_INCREMENT,
+	name VARCHAR(255) NOT NULL,
+	phone_number VARCHAR(255) NOT NULL,
+	address VARCHAR(255) NOT NULL, 
+	payment_method VARCHAR(10),
```
Comment (ToluEnny24) [inline submissions/josephOlasimbo/database/init.sql:38]: This should be ENUM too.

File: submissions/josephOlasimbo/database/init.sql [hunk 6]
```diff
@@ -0,0 +1,48 @@
...
+    PRIMARY KEY (id)
+);
+
+-- CREATE THE orders TABLE
+CREATE TABLE orders (
+id INT NOT NULL AUTO_INCREMENT,
+vendor_id INT NOT NULL,
+customer_id INT NOT NULL,
+PRIMARY KEY (id)
+);
```
Comment (ToluEnny24) [inline submissions/josephOlasimbo/database/init.sql:48]: FOREIGN KEYS are missing here too
customer_id should be optional

**Review by ToluEnny24 — CHANGES_REQUESTED** [review]
Please review all requested changes. @joejoba

**Review by Mayowa2020 — APPROVED** [review]
Well done @joejoba! Approved...

**Pull request merged**